appveyor.yml

```diff
@@ -20,22 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```

Comment:
Ok, so this just worked the second time. The sporadic nature of this issue has made things difficult. If you have any details about the first re-rendering that you can share, that would be very helpful (e.g. packages in your environment, log, etc.).